### DISCUSSION TITLE
Image resizing and post-processing

### DISCUSSION THREAD

**Discussion by pkucmus** [saleor/saleor#11790] [Ideas]
## Thumbnail size presets

Saleor currently has a set of [predefined image sizes](https://github.com/saleor/saleor/blob/main/saleor/thumbnail/__init__.py#L4). These would probably work for most of the cases but I wonder if we could have them configurable on the "shop" level, i.e. I'd like to be able to configure the thumbnail sizes to 40, 100 and 200. A maximum of 8 available size settings would be enough (not to explode the cache).

## Media post-processing

Additionally (this addresses a different case), I'd like to propose `MEDIA` related webhooks - those would allow us to deal with cases like adding a blurhash, watermarking and also resizing.

**Comment by timuric** [comment]
media related webhooks issue posted https://github.com/saleor/saleor/issues/11750
